### DISCUSSION TITLE
Welcome to docker-lidarr-extended Discussions!

### DISCUSSION THREAD

**Discussion by RandomNinjaAtk** [RandomNinjaAtk/docker-lidarr-extended#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by bugleboyLP** [comment]
Hello, I just installed the Lidarr-extended docker container and i have lidarr running but don't see any of the integrated items mentioned in the description such as the deemix/tidal download clients or indexers.  do I need to add those as additional docker containers or have i missed something in the configuration?

**Reply by RandomNinjaAtk** [reply]
This is a automation bash script that works in the background, you won't see it in any web interface, if you have configured everything properly, look in log folders, and docker container log to see what is going on.

You can see relevant logs from the Lidarr web interface, navigate to: system -> log files.

The idea is for it to just work in the background, and you use Lidarr normally. You'll just need to check to make sure its functioning. Once you understand how it works, you'll probably like it.

**Reply by nayab9** [reply]
It wasn't entirely obvious to me how to configure the script in Lidarr, at least for Tidal.

After starting the container with all the appropriate parameters for Tidal (crucially for me it would be, `-e dlClientSource=Tidal` and `-e tidalCountryCode=BR`) - a logfile is generated showing the output of the scripts:

`extended_script_run_*.txt`

You can find this within Lidarr GUI (System -> Logs), or just CLI log dir if preferred.

When the script attempts to setup Tidal, it will give you 5 minutes to go to the URL provided with a device code, to allow access for the script as a client.
After I did this step, it was fully automated from there.
I have not added anything else to Lidarr - clean fresh install with only Tidal as the source.

Seems to work pretty great so far.

Very much appreciate this container @RandomNinjaAtk , works fantastic on unRAID
